snuba/cli/subscriptions_verifier.py

```diff
@@ -0,0 +1,70 @@
+import signal
+from typing import Any
+
+import click
+from arroyo import configure_metrics
+
+from snuba import environment
+from snuba.environment import setup_logging, setup_sentry
+from snuba.subscriptions.verifier import build_verifier
+from snuba.utils.metrics.wrapper import MetricsWrapper
+from snuba.utils.streams.metrics_adapter import StreamMetricsAdapter
+
+
+@click.command()
+@click.option(
+    "--consumer-group",
+    default="snuba-subscription-verifier",
+    help="Consumer group used for consuming the commit log topic.",
+)
+@click.option(
+    "--orig-result-topic", required=True, type=str, help="Original result topic"
+)
+@click.option("--new-result-topic", required=True, type=str, help="New result topic")
+@click.option(
+    "--threshold-sec",
+    default=30,
```

Comment:
I think you should give yourself at least a minute considering the jitter we apply is up to one minute